documentation/blog/2023-10-01-whats-new-in-svelte-october-2023.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "What's new in Svelte: October 2023"
+description: "Reactions to Runes and SvelteKit +server fallbacks"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Svelte 5 isn't out yet (you can, however, [preview it now](https://svelte-5-preview.vercel.app/)), but that doesn't mean we don't get a sneak peek! Most notably are [Runes](https://svelte.dev/blog/runes) - a simpler way to manage reactive variables in Svelte code. There's lots of links the showcase section for deeper dives on all things Runes, but let's talk about what else been released this month...
+
+## What's new in Svelte & Language Tools
+- [Svelte 4.2.1](https://github.com/sveltejs/svelte/blob/master/packages/svelte/CHANGELOG.md#421) was released with a bunch of fixes to HTML, CSS and sourcemap compilation
+- [The latest version of the Svelte language tools](https://github.com/sveltejs/language-tools/releases/tag/extensions-107.11.0) [enhances component references](https://github.com/sveltejs/language-tools/pull/2157) in the "Find All References" command and [adds fallback handling to auto-types](https://github.com/sveltejs/language-tools/issues/2156) (like those found in SvelteKit's `+server.js` files)
```

Comment:
It would also be good to mention that the latest language tools also fixes a persistent issue where the auto-generated types would be missing when restarting a project: https://github.com/sveltejs/language-tools/pull/2160